### PR TITLE
refractor(df/locale):shares module

### PR DESCRIPTION
**Pull request by abhishekkumar08**
## Description
Update dateformats and language with centralized data in shares
module

## Related issues and discussion
#1213 

## Checklist
Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] If you have multiple commits please combine them into one commit by squashing them.

- [x] Read and understood the contribution guidelines at `web-app/.github/CONTRIBUTING.md`.

### CODE REVIEW

**Comment by abhishekkumar08** [comment]
@karantakalkar @BLasan Please review this PR. Thanks.

**Review by BLasan — APPROVED** [review]
@abhishekkumar08 LGMT! We'll wait until @karantakalkar  approve these changes

**Review by Jov03 — APPROVED** [review]

**Pull request merged**
